src/instruments/src/MCDU/Pages/PagesContainer.tsx

```diff
@@ -27,6 +27,8 @@ import IdentPage from './FMGC/Ident/Ident';
 import { PAGE_KEYS } from '../Components/Buttons';
 import FuelPred from './FMGC/Fuel/FuelPred';
 import FplanHolder from './FMGC/Fplan/Fplan';
+import RadnavPage from './FMGC/RadNav/Radnav';
+import { addScratchpadMessage } from '../redux/actions/scratchpadActionCreators';
```

Comment:
This is the incorrect way to be using redux hooks. this action creator by itself won't do anything you need to dispatch it. Also, this is the pages container but I wouldn't use it as a redux container, better you do that inside of `Radnav.tsx`